cmd/skywire-cli/commands/vpn/root.go

```diff
@@ -0,0 +1,23 @@
+package clivpn
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/skycoin/skywire-utilities/pkg/logging"
+)
+
+var (
+	logger  = logging.MustGetLogger("skywire-cli")
+	path    string
+	pkg     bool
+	ver     string
```

Comment:
Why are we putting these into the root command? Maybe I am missing some feature here of cobra, but they seem unsued. If they are connected to the variables we are using in the submodules, why are we not declaring them there?